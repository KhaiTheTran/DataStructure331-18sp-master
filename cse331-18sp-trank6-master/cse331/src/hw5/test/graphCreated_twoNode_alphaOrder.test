# Test graphCreated_twoNode_alphaOrder

# Create a graph
CreateGraph graphCreated_twoNode_alphaOrder

# Add two nodes in backward order
AddNode graphCreated_twoNode_alphaOrder yy
AddNode graphCreated_twoNode_alphaOrder xx

# Print the nodes in the graph and the outgoing edges from xx,
# and outgoing from yy
ListNodes graphCreated_twoNode_alphaOrder
ListChildren graphCreated_twoNode_alphaOrder xx
ListChildren graphCreated_twoNode_alphaOrder yy