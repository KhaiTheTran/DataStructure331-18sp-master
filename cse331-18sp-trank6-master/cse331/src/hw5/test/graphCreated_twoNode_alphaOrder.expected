# Test graphCreated_twoNode_alphaOrder

# Create a graph
created graph graphCreated_twoNode_alphaOrder

# Add two nodes in backward order
added node yy to graphCreated_twoNode_alphaOrder
added node xx to graphCreated_twoNode_alphaOrder

# Print the nodes in the graph and the outgoing edges from xx,
# and outgoing from yy
graphCreated_twoNode_alphaOrder contains: xx yy
the children of xx in graphCreated_twoNode_alphaOrder are:
the children of yy in graphCreated_twoNode_alphaOrder are: